class.jetpack-connection-banner.php

```diff
@@ -25,6 +25,39 @@ private function __construct() {
 	}
 
 	/**
+	 * Checks whether the connection banner A/B test should be ran.
```

Comment:
Or, could we link to do documentation on how to read gmmktime? 😝 

I'm kidding. A comment to explain is more than reasonable.